### PR TITLE
adding rhacs to cloud docs

### PR DESCRIPTION
**Pull request by TehreemNisa**

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-316-6242ee6a`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-316-f4257968`

File: content/sre/security/rhacs/02-permissions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Roles and Users
+
+### Users
+
+We divide the rhacs users into 2 groups according to the interaction method:
+
+1. human users and
```
Comment (usamaahmadkhan) [inline content/sre/security/rhacs/02-permissions.md]: First letter should always be Capital

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-316-47ae8bf2`

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

File: content/sre/security/rhacs/02-permissions.md (hunk 1 above)
Comment (TehreemNisa) [reply content/sre/security/rhacs/02-permissions.md]: updated

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-316-4e645b7f`

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.246`

**Pull request merged**